dev/com.ibm.ws.kernel.boot_fat/publish/files/versionlessServer.xml

```diff
@@ -0,0 +1,25 @@
+<!--
+    Copyright (c) 2017 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License 2.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-2.0/
+    
+    SPDX-License-Identifier: EPL-2.0
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server>
+	<include location="../fatTestPorts.xml"/>
+    <config monitorInterval="1s" />
+    
+    <featureManager>
+        <platform>jakartaee-10.0</platform>
+        <platform>microProfile-4.0</platform>
```

Comment:
Since these conflict with each other what is this supposed to do?  microProfile-4.0 is EE 8 based not EE 10 based.

Reply:
fixing in a new pr -> https://github.com/OpenLiberty/open-liberty/pull/29084